scheduled-jobs/scanning/art-notify/Jenkinsfile

```diff
@@ -17,6 +17,11 @@ node() {
                         trim: true,
                         defaultValue: "#team-art"
                     ),
+                    booleanParam(
+                        name: 'TAG_RELEASE_ARTISTS',
```

Comment:
why add this? if the purpose is not tagging during testing, maybe a dry_run is better

Reply:
ack - updated